### PR TITLE
3.0-ify create app and element pages

### LINKED ISSUE #2516: 3.0-ify app/3.0/docs/tools/create-element-polymer-cli.md

### PR DESCRIPTION
**Pull request by katejeffreys**
Fixes #2516.

### CODE REVIEW

File: app/3.0/docs/tools/create-app-polymer-cli.md [hunk 1]
```diff
@@ -74,29 +83,33 @@ application-specific elements should be defined in the `src` directory, at the s
...
    Don't use `polymer init` to create an element project inside your app project.
 
 ## Manage dependencies in an application project {#dependencies}
 
-Applications should use relative URLs to import other source files and dependencies. But because 
-applications are served independently, they can properly reach into the dependencies directory for 
-dependencies.
+Import your dependencies with module specifiers:
+
+src/my-app.js {.caption}
```
Comment (arthurevans) [inline app/3.0/docs/tools/create-app-polymer-cli.md]: Diagram above shows this as `src/my-app/my-app.js`?

File: app/3.0/docs/tools/create-app-polymer-cli.md [hunk 2]
```diff
@@ -74,29 +83,33 @@ application-specific elements should be defined in the `src` directory, at the s
...
+
+```
+import '@polymer/polymer/polymer-element.js';
+import 'my-el/my-el.js';
```
Comment (arthurevans) [inline app/3.0/docs/tools/create-app-polymer-cli.md]: Import for a local resource must start with ./ ../ or / ... So this should be `../my-el/my-el.js` or `/src/my-el/my-el.js`

File: app/3.0/docs/tools/create-app-polymer-cli.md [hunk 3]
```diff
@@ -74,29 +83,33 @@ application-specific elements should be defined in the `src` directory, at the s
...
+```
 
-    <!-- from a 'src/some-application.html' file in your application -->
-    <link rel="import" href="../bower_components/polymer/polymer.html">
\ No newline at end of file
+Polymer CLI rewrites module specifiers to paths when you build your app. See [Build for production](/{{{polymer_version_dir}}}/toolbox/build-for-production) for more information.
```
[the remark this answers is not in the document]
Comment (katejeffreys) [reply app/3.0/docs/tools/create-app-polymer-cli.md]: Thanks, this explanation is great!

File: app/3.0/docs/tools/create-app-polymer-cli.md (hunk 1 above)
Comment (katejeffreys) [reply app/3.0/docs/tools/create-app-polymer-cli.md]: - [ ]

**Pull request merged**